dotnet/src/extensions/Azure/Handlers/bin/Debug/net5.0/GeneXus.Deploy.AzureFunctions.Handlers.deps.json

```diff
@@ -0,0 +1,2384 @@
+{
+  "runtimeTarget": {
```

Comment:
Seems strange to me, that this File is in "Debug" Folder.

Reply:
It shouldn't have been pushed. I'll remove it.